torch/csrc/lazy/core/ir.cpp

```diff
@@ -6,6 +6,19 @@ C10_DEFINE_bool(ltc_enable_dynamic_shapes, false, "Whether dynamic shape is enab
 namespace torch {
 namespace lazy {
 
+hash_t OperandHashes(const OpList& operands, const hash_t& seed, bool bakeInSizes) {
```

Comment:
we should align this change with @shunting314's proposed changes to fix what @JackCaoG was asking us to do, to move the hash_with_sizes stuff strictly inside TsNode